src/views/LoadManagementConfig.vue

```diff
@@ -432,6 +480,19 @@ export default {
 			});
 			return myChargePoint;
 		},
+		getHcSourceIdOptions() {
+			var options = [
+				{
+					value: null,
+					text: "von openWB berechnen",
+				},
+			];
+			options.push(...this.counterOptions);
+			return options;
+		},
```

Comment:
Hier würde ich mit Gruppen arbeiten, um den ersten Eintrag etwas abzuheben.
```suggestion
		getHcSourceIdOptions() {
			let options = [
				{
					value: null,
					text: "von openWB berechnen",
				},
			];
			let groups = [
				{
					label: "Eingerichtete Zähler-Komponenten",
					options: [...this.counterOptions);
				}
			]:
			return { options: options, groups: groups };
		},
```